flake.nix

```diff
@@ -90,14 +52,8 @@
               [
                 pkgs.corepack
                 pkgs.vsce
-                pkgs.nodejs
+                pkgs.nodejs_20
 
-                # https://github.com/NixOS/nixpkgs/pull/251418
-                (pkgs.pre-commit.overrideAttrs (previousAttrs: {
```

Comment:
Isn't this change removing the `pre-commit` package entirely?

Reply:
oops, good catch ty